samples/java_springboot/07.using-adaptive-cards/src/main/java/com/microsoft/bot/sample/usingadaptivecards/AdaptiveCardsBot.java

```diff
@@ -69,7 +69,7 @@
             if (!member.getId().equals(turnContext.getActivity().getRecipient().getId())) {
                  turnContext.sendActivity(
                      String.format("Welcome to Adaptive Cards Bot %s. %s", member.getName(), welcomeText)
-                 ).join();
+                 );
```

Comment:
This is not the correct implementation. As mentioned in Slite, you can follow [this pattern](https://github.com/microsoft/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java#L58-L75).

The join should be removed, but the `CompletedFuture` also should be managed.